services/google_search_service/db_updater_test.go

```diff
@@ -0,0 +1,106 @@
+package google_search_service
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/bxcodec/faker/v3"
```

Comment:
Small thing, `github.com/bxcodec/faker/v3` is alo an external lib, is it correct @gutakk 😀

Reply:
Exactly, fixed in f7c2c7bd294695117f9990e97ec4d91c96eebe35.
For another old files I will refactor to follow our convention in https://github.com/gutakk/go-google-scraper/issues/33